esmvaltool/cmorizers/obs/cmorize_obs_mls.py

```diff
@@ -0,0 +1,341 @@
+"""ESMValTool CMORizer for MLS data.
+
+Tier
+    Tier 3: restricted dataset.
+
+Source
+    https://disc.gsfc.nasa.gov/datasets/ML2RHI_004/summary
+    https://disc.gsfc.nasa.gov/datasets/ML2T_004/summary
+
+Last access
+    20200203
+
+Download and processing instructions
+    Select "Data Access" -> "Subset/Get Data" -> "Get Data" and follow the
+    "Instructions for downloading". All *.he5 files need to be saved in the
+    $RAWOBS/Tier3/MLS directory, where $RAWOBS refers to the RAWOBS directory
+    defined in the user configuration file. The temperature fields are
```

Comment:
I would add something like: "Apply this procedure to both links provided above. The temperature fields..."